src/pages/STG_PanelAlloBatch.page

```diff
@@ -1,18 +1,19 @@
 <apex:page controller="STG_PanelAlloBatch_CTRL" >
-    <c:STG_PageHeader sectionLabel="{!$Label.stgNavSystem}" pageLabel="{!$Label.stgNavRollupAlloBatch}" />
-    <p>{!$Label.stgHelpBatchAlloRollup}</p>
-
-    <apex:form id="form" styleClass="form-horizontal">   
-        <apex:pageMessages id="pageMessages"/>
-        <apex:pageBlock id="pbRelBatch" mode="maindetail">
-            <div class="button-block">
-                <apex:commandButton value="{!$Label.stgBtnRunBatch}" status="statusLoad" action="{!runBatch}" disabled="{!isRunningBatch}" immediate="true" rerender="pbRelBatch" styleClass="btn-sm btn-primary" />
-            </div>
-              
-            <apex:outputPanel rendered="{!isRunningBatch}">
-                <c:UTIL_JobProgress strBatchComponentLabel="{!$Label.stgLabelAlloBatchProgress}" cNumberOfJobs="1" bRerenderOnComplete="true" />
-            </apex:outputPanel>
-            
-        </apex:pageBlock>
+    <apex:form id="form" styleClass="slds-m-around--x-large">
+        <c:STG_PageHeader sectionLabel="{!$Label.stgNavBulkProcesses}" pageLabel="{!$Label.stgNavRollupAlloBatch}" />
+        <div class="slds-text-body--small slds-m-around--medium">{!$Label.stgHelpBatchAlloRollup}</div>
+        <c:UTIL_PageMessages />
+        <div class="slds-grid slds-grid--align-center slds-grid--vertical-align-center slds-p-around--large">
+            <apex:commandButton value="{!$Label.stgBtnRunBatch}" status="statusLoad" action="{!runBatch}" disabled="{!isRunningBatch}" immediate="true" rerender="form" styleClass="slds-button slds-button--small slds-button--neutral" />
```

Comment:
button always is disabled for me, even after refreshing the page, so I can never run this batch!